.github/components.yml

```diff
@@ -199,6 +199,11 @@ IE_Tests:
     - IR_FE
 
 MO:
+  revalidate:
+    - PyTorch_FE
+    - TF_FE
+    - ONNX_FE
+    - PDPD_FE
```

Comment:
do  you mean that `convert_model` must revalidate frontends?
how PDPD is affected by changes in MO / OVC?

Reply:
MO is responsible for frontend configuration. It invoked FE API for this configuration. So if we change something in this common logic incorrectly, we can break conversion of some FW models, for example.

Reply:
If PDPD FE does not have e2e tests in precommit, I can remove it.

Reply:
MO is built on top of frontends => frontends does not depend on MO, does they?

Reply:
ok, let's keep only those FEs which rely on OVC / MO a lot

Reply:
> MO is built on top of frontends => frontends does not depend on MO, does they?

This dependency is unfortunately created due to these integration tests.

Reply:
PR and TF are left.